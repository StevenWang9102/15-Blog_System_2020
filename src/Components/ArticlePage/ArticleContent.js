import React from "react";
import { createUseStyles } from "react-jss";

const useStyles = createUseStyles({
  content: {
    textAlign: "left",
    marginLeft: "10px"
  }
});


export const ArticleContent = props => {
  const Content = ({ children }) => {
    const classes = useStyles();
    return (
      <div className={`row article-content ${classes.content}`}>{children}</div>
    );
  };

  return (
    <div className='container page'>
      <Content>
        <div className='col-md-12 article-detail'>
          {props.currentArticleDetails.body}
        </div>
      </Content>
      <hr />
    </div>
  );
};
