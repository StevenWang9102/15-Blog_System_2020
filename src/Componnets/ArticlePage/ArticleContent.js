import React from "react";

export const ArticleContent = props => {
  return (
    <div className='container page'>
    <div className='row article-content'>
      <div className='col-md-12 article-detail'>
        {props.currentArticleDetails.body}
      </div>
    </div>
    <hr />
  </div>
  );
};