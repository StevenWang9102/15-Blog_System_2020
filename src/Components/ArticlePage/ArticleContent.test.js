import React from 'react';
import { ArticleContent } from './ArticleContent';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<ArticleContent currentArticleDetails={{}}/>).toJSON();
  expect(tree).toMatchSnapshot();
});