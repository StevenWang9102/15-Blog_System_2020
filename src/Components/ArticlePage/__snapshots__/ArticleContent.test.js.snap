// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container page"
>
  <div
    className="row article-content content-0-2-1"
  >
    <div
      className="col-md-12 article-detail"
    />
  </div>
  <hr />
</div>
`;
